Tests/Resources/app/config/cmf_routing.phpcr.yml

```diff
@@ -3,4 +3,4 @@ cmf_routing:
         persistence:
             phpcr:
                 enabled: true
-                route_basepath: /test/routing
+                route_basepaths: /test/routing
```

Comment:
should this be an `[ 'array' ]` ? I know that we probably support scalars, but I think its better to treat a plural as a plural.